### DISCUSSION TITLE
GitHub  use

### DISCUSSION THREAD

**Discussion by Smithaaaa** [EddieHubCommunity/support#384] [Q&A] [answered]
Hi,I am new,to GitHub ,I need ur help, how to join this GitHub community to do projects,please help me

**Comment by eddiejaoude** [comment, marked as answer]
Hi, we would love you to join the community, can you create an issue here please and we can send you an invite  https://github.com/EddieJaoudeCommunity/support/issues/new?assignees=eddiejaoude%2C+mikeysan&labels=invite+me+to+the+organisation&template=invitation.md&title=Please+invite+me+to+the+GitHub+Community+Organisation

**Comment by AllanRegush** [comment]
@Smithaaaa Checking in. Have you made a ticket to join the organization? We would love to have you. 😄
